app/src/androidTest/java/com/github/houseorganizer/houseorganizer/LoginActivityTest.java

```diff
@@ -17,22 +17,11 @@
 import org.junit.runner.RunWith;
 
 @RunWith(AndroidJUnit4.class)
-public class MainActivityTest {
+public class LoginActivityTest {
 
     @Rule
-    public ActivityScenarioRule<MainActivity> testRule = new ActivityScenarioRule<>(MainActivity.class);
+    public ActivityScenarioRule<LoginActivity> testRule = new ActivityScenarioRule<>(LoginActivity.class);
+    
 
-    // Test that when the button is clicked an intent is fired and the intent contains the username
-    @Test
-    public void clickButton() {
```

Comment:
I would suggest to leave the test I added since it tests if the sign in button works

`// Test that when the sign in button is clicked an intent is fired
    @Test
    public void clickSignInButtonFiresIntent() {
        Intents.init();
        onView(withId(R.id.sign_in_button)).perform(click());
        intended(toPackage("com.github.houseorganizer.houseorganizer"));
        Intents.release();
    }`

Reply:
The thing is it does not pass anymore as there is no intent fired when the sign-in is done for now. It will be useful again once we implement the compatibility with the main activity.